rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec

```diff
@@ -16,6 +16,8 @@ Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(foreman-tasks-core) >= 0.1
 Requires: %{?scl_prefix}rubygem(foreman-tasks-core) < 1.0
+Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core) >= 1.1
```

Comment:
there's no such rex core, rex core was not released as there was no change in its codebase

Reply:
please ignore me, the package is there, it's just not packaged for some reason

Reply:
it seems it was bumped by https://github.com/theforeman/foreman-packaging/commit/2028d1d18eee68c471c1e9eafdd082ec11cd8a23 but never built, could someone who merges this also trigger the build pls?

Reply:
resolved with @ekohl package is built now